models/comments.model.js

```diff
@@ -0,0 +1,22 @@
+const db = require("../db/connection.js");
+
+
+exports.insertCommentToArticle = (newComment, article_id) => {
+    const { username, body } = newComment;
+
+    if (typeof body !== "string") {
+        return Promise.reject({
+            status: 400,
+            msg: "invalid input",
+        })
+    }
```

Comment:
our db.query will also return an error when passed the wrong data type for an argument, so by using that psql error we can avoid having to use a custom error handler. Taking advantage of built in functionality is almost always a good thing!